### ISSUE TITLE
Clean up /web/examples/howto/web-in-60/static-content.rst

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7049]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7049|
|Type|defect|
|Created|2014-03-24 07:14:01Z|
|Branch|https://github.com/twisted/twisted/tree/clean-up-web-static-docs-7049|

This corrects a rendering error and makes the source file easier to read.

Attachments:

* [t7049.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c9a/c9a6a809979b1b391e87ed8939e115cac57de98d/8d0cc3fd8922ce684593c94d7d9b8241cabd9b01.diff) (3024 bytes) - added by Alex on 2014-03-24 07:14:20Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7049 7049
type__defect defect
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_clean_up_web_static_docs_7049 branches/clean-up-web-static-docs-7049
branch_author__Alex_hawkowl Alex,hawkowl
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1395645241000000 1395645241000000
changetime__1395647439000000 1395647439000000
version__None None
owner__ 
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

(In [41971]) Branching to clean-up-web-static-docs-7049.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

Alex, your changes there are fine. As I've done further changes in this branch, I'm putting my set up for review.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl commented|
|-|-|

Putting my changes up for review.

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_7"></a>|Alex commented|
|-|-|

Just reviewed your 3 commits on this branch. Assuming `::` renders with correct syntax highlighting like `.. code-block:: python` does, then this looks good to me.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl set status to `closed`|
|-|-|

(In [41977]) Merge clean-up-web-static-docs-7049

Authors: Alex, hawkowl
Reviewers: hawkowl, Alex
Fixes: [#7049](https://github.com/twisted/twisted/issues/7049)

Fixes whitespace and syntax in web-in-60/static-content.rst.
